en/modals.ftl

```diff
@@ -4,13 +4,15 @@
 
 ## Modal for generating a custom mask
 
+modal-custom-alias-picker-tip = Tip: To create a custom mask anytime, make it up on the spot. If you use your Relay domain, it’ll work, even if you didn’t generate it here first.
+
 modal-custom-alias-picker-heading-2 = Create a new custom mask
 modal-custom-alias-picker-warning-2 = All you need to do is make up and share a unique mask that uses your custom subdomain — the mask will be generated automatically. Try “shop@customsubdomain.mozmail.com” next time you shop online, for example.
 modal-custom-alias-picker-form-heading-2 = Or, create a custom mask manually
-modal-custom-alias-picker-form-prefix-label-2 = Enter email mask prefix
+modal-custom-alias-picker-form-prefix-label-v3 = Enter the text that goes before the @ symbol:
```

Comment:
You may need to check with other devs on the convention to indicate version of a string. Looks like `-3` might be the way Relay team handles it so it stays consistent with the strings around this one.